client/extensions/woocommerce/app/settings/payments/payments-offline.js

```diff
@@ -8,23 +8,23 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import Card from 'components/card';
-import SectionHeader from 'components/section-header';
+import ExtendedHeader from '../../../components/extended-header';
 
 class SettingsPaymentsOffline extends Component {
 
 	render() {
 		const { translate } = this.props;
 		return (
 			<div>
-				<SectionHeader label={ translate( 'Offline payment methods' ) } />
-				<Card>
-					{
+				<ExtendedHeader
+					label={ translate( 'Offline payment methods' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Payment Methods' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).